imap_processing/glows/tests/test_glows_decom.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from imap_processing.glows.l0 import glows_decom
+
+logging.basicConfig(level=logging.ERROR)
+
+
+@pytest.fixture()
+def decom_test_data():
+    """Read test data from file"""
+    current_directory = Path(__file__).parent
+    packet_path = f"{current_directory}/glows_test_packet_20230920_v00.pkts"
+    data_packet_list = glows_decom.decom_packets(packet_path)
+    return data_packet_list
+
+
+def test_glows_decom_count(decom_test_data):
+    expected_hist_len = 61
+    expected_de_len = 311
```

Comment:
Is this the number of histogram packets in the packet file? A quick comment or a change of variable name to `expected_num_hist_packets` or something like that would help I think.

Reply:
Yes, it is. I can update it to be more clear